libraries/plugins/peerplays_sidechain/bitcoin_utils.cpp

```diff
@@ -677,4 +759,25 @@ bytes add_signatures_to_unsigned_tx(const bytes &unsigned_tx, const std::vector<
    return ret;
 }
 
+std::string get_weighted_multisig_address(const std::vector<std::pair<std::string, uint64_t>> &public_keys) {
+   std::vector<std::pair<fc::ecc::public_key, uint64_t>> key_data;
+   for (auto p : public_keys) {
+      fc::ecc::public_key_data kd;
+      for (uint32_t i = 0; i < p.first.length(); i += 2) {
+         std::string byte_str = p.first.substr(i, 2);
+         kd.data[i / 2] = (char)std::strtol(byte_str.c_str(), NULL, 16);
+      }
```

Comment:
Bitcoin public keys in configuration file are HEX encoded. I changed this part to read them properly.
However, The final result is invalid address. Eg, for default keys we are using, we get invalid address:
```
03456772301e221026269d3095ab5cb623fc239835b583ae4632f99a15107ef275
02d67c26cf20153fe7625ca1454222d3b3aeb53b122d8a0f7d32a3dd4b2c2016f4
025f7cfda933516fd590c5a34ad4a68e3143b6f4155a64b3aab2c55fb851150f61
0228155bb1ddcd11c7f14a2752565178023aa963f84ea6b6a052bddebad6fe9866
037500441cfb4484da377073459511823b344f1ef0d46bac1efd4c7c466746f666
02ef0d79bfdb99ab0be674b1d5d06c24debd74bffdc28d466633d6668cc281cccf
0317941e4219548682fb8d8e172f0a8ce4d83ce21272435c85d598558c8e060b7f
0266065b27f7e3d3ad45b471b1cd4e02de73fc4737dc2679915a45e293c5adcf84
023821cc3da7be9e8cdceb8f146e9ddd78a9519875ecc5b42fe645af690544bccf
0229ff2b2106b76c27c393e82d71c20eec32bcf1f0cf1a9aca8a237269a67ff3e5
024d113381cc09deb8a6da62e0470644d1a06de82be2725b5052668c8845a4a8da
03df2462a5a2f681a3896f61964a65566ff77448be9a55a6da18506fd9c6c051c1
02bafba3096f546cc5831ce1e49ba7142478a659f2d689bbc70ed37235255172a8
0287bcbd4f5d357f89a86979b386402445d7e9a5dccfd16146d1d2ab0dc2c32ae8
02053859d76aa375d6f343a60e3678e906c008015e32fe4712b1fd2b26473bdd73
Bitcoin RPC call importaddress with body {"jsonrpc": "1.0", "id":"importaddress", "method": "importaddress", "params": ["bc1q6a5l7jw8q9kmvcv3yjfyncp0fk69q0vcwslty5e800xd4ugm8s9qjag62c"] } failed with reply '{"result":null,"error":{"code":-5,"message":"Invalid Bitcoin address or script"},"id":"importaddress"}

The address we get is:
bc1q6a5l7jw8q9kmvcv3yjfyncp0fk69q0vcwslty5e800xd4ugm8s9qjag62c
```

Reply:
It is a mainnet address,  you are trying to publish it in regtest network.Mainnet addresses are started with `bc`,  testnet and regtest addresses are started with `tb`